Workbooks/Images/Logos/ubiquiti.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="75" height="75" viewBox="0 0 75 75" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M64.1229 41.7575L65.5176 37.2065V6.0375C49.217 6.0375 47.5194 12.2888 47.5194 18.2567V52.4861C47.5194 54.2587 47.3471 56.1397 46.9887 57.9817C55.6322 56.4078 61.2709 51.065 64.1229 41.7561" fill="#0D0D0D"/>
+<path d="M22.3975 13.1543H18.7953V9.59658H22.3975V13.1543ZM25.9954 18.5304H22.3975V14.9699H25.9954V18.5304ZM22.3766 23.0077H18.8162V19.4487H22.3766V23.0077ZM18.7773 28.345H15.2182V24.7859H18.7773V28.345ZM11.598 9.59658H8V6H11.598V9.59658Z" fill="#39A2E1"/>
+<path d="M15.2001 31.9013H18.7939V39.8293H22.3975V26.5654H25.9954V52.4875C25.9954 64.7873 36.7616 70.7205 36.7616 70.7205C18.6772 70.7205 8 59.7599 8 45.1735V15.8423H11.598V35.4604H15.2001V31.9013Z" fill="#39A2E1"/>
+<path d="M39.2857 59.903C34.4847 59.6849 30.7909 58.4666 27.9292 56.7037C29.8671 65.1152 37.1144 69.2466 37.4645 69.4425L39.6011 70.6205C55.9489 69.5286 65.5176 58.975 65.5176 45.1749V42.184C61.6321 54.8616 52.5885 60.5017 39.2857 59.903Z" fill="url(#paint0_linear)"/>
+<defs>
+<linearGradient id="paint0_linear" x1="27.9292" y1="56.4022" x2="65.5162" y2="56.4022" gradientUnits="userSpaceOnUse">
```

Comment:
Invalid Logo, Please follow the guidelines provided here: https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors

company logo adhering to the following requirements in the 'Logo' folder
Logo needs to be in SVG format and under 5 Kb
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
Do not use xlink:href - use inline instead
Do not use title tag
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square
SVG code file is formatted well for readability

Reply:
@vu-socprime : Please find the valid logo file below. Please use this svg file for logo.
[ubiquiti.zip](https://github.com/Azure/Azure-Sentinel/files/6180053/ubiquiti.zip)

Thanks,
Aditi Maheshwari

Reply:
Updated Logo